README.md

```diff
@@ -1,2 +1,34 @@
-# loopback-sdk-ios-codegen
-Code generation tool for LoopBack iOS SDK
+# LoopBack iOS SDK CodeGen
+
+The iOS CodeGen command line tool generates iOS client side LoopBack Model representations
+in Objective-C by looking into the specified server application.
+
+## How to Use
+
+The following is an example of the usage applied to the test server provided under this directory:
+
+ 1. Run `npm install` in this directory to initialize the tool `bin/codegen-ios`.
+ *  Run `pushd test-env; npm install; popd` to initialize the test server.
```

Comment:
Thanks for pointing it out!  I've updated the master `package.json` and removed the child.